vault/dendron.release.2021-11-09.md

```diff
@@ -0,0 +1,92 @@
+---
+id: EjPEcAkZIDgzHaVMk4teA
+title: Release Notes (version 0.68)
+desc: ''
+updated: 1636484692932
+created: 1636478484110
+---
+
+Dendron 0.68 has sprouted  🌱
+
+Take a look at the latest [Dendron wiki](https://wiki.dendron.so), and you'll see some new features directly from these highlights:
+
+* Local TOC in the right-hand nav for each page in the published docs
+* Newly combined search bar that can go directly to a note, or search note contents using the `?` prefix
+
+`Dendron: Show Preview` used to add some extra latency when rendering the preview and also made features like copy/paste unstable. This change migrates the old iframe-based preview to be a native webview. It brings the following changes:
```

Comment:
i would move the `Dendron: Show Preview` notes from here to the changelog. 
for the release notes, our audience is a technical person that might not be familiar with web development or vscode so avoid using terms like iframe or webview in the release.